src/Microsoft.AspNet.Authentication.Twitter/TwitterOptions.cs

```diff
@@ -82,7 +44,11 @@ public string DisplayName
         /// <summary>
         /// Gets or sets the <see cref="ITwitterEvents"/> used to handle authentication events.
         /// </summary>
-        public ITwitterEvents Events { get; set; }
+        public new ITwitterEvents Events
```

Comment:
Yep we brought back Event inheritance (but without generics), but since it allowed me to actually make Twitter/OIDC/OAuth share a base class and same Invoke path code, I think its worth it now :)